ivy_tests/test_ivy/test_functional/test_core/test_creation.py

```diff
@@ -405,25 +404,23 @@ def test_from_dlpack(
     dtype, x = dtype_and_x
     helpers.test_function(
         input_dtypes=dtype,
-        as_variable_flags=False,  # can't convert variables
+        as_variable_flags=[False],  # can't convert variables
```

Comment:
changing lines like this is okay (perhaps it runs more efficiently when passed as a list), but I think we should still be able to handle the case if this was passed as `False`. Are we still able to handle such cases?

Reply:
We used to convert these into lists in the test functions, this adds some overhead which isn't necessary (we check if it's a list, if not we create one), also it's more intuitive to pass list to `flags`, I could add it back if that makes things simpler.